### PR TITLE
Fix/move nvm to licode deps

### PR DESCRIPTION
**Pull request by lodoyun**
Moved nvm to licode deps
This makes it possible for Licode to find nvm in more complex configurations

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,10 +40,8 @@ before_install:
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then   ./scripts/installMacDeps.sh --unattended --disable-services --use-cache;    fi
 
   # LINUX
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then ./scripts/installTravisDeps.sh && export CC="/usr/bin/gcc-5" && export CXX="/usr/bin/g++-5" ; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then ./scripts/travisInstallDeps.sh && export CC="/usr/bin/gcc-5" && export CXX="/usr/bin/g++-5" ; fi
```
Comment (jcague) [inline .travis.yml]: Why do we change the name here? we do have a kind of pattern with install[System]Deps.sh, am I right?

File: scripts/travisInstall.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+SCRIPT=`pwd`/$0
```
Comment (jcague) [inline scripts/travisInstall.sh]: don't we want to use `set -e` here?

File: scripts/travisScript.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+SCRIPT=`pwd`/$0
```
Comment (jcague) [inline scripts/travisScript.sh]: `set -e`?

File: .travis.yml [hunk 4]
```diff
@@ -41,10 +40,8 @@ before_install:
...
 install:
-  - ./scripts/installErizo.sh && ./scripts/installNuve.sh && npm install
+  - ./scripts/travisInstall.sh
```
Comment (jcague) [inline .travis.yml]: +1 to have a single script here

File: scripts/checkNvm.sh [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+CHECK_SCRIPT=$(readlink -f "$0")
+CHECK_SCRIPT_PATH=$(dirname "$CHECK_SCRIPT")
+LINKED_DIR=$CHECK_SCRIPT_PATH/../build/libdeps/nvm
+export NVM_DIR=$(readlink -f "$LINKED_DIR")
+echo "Checking dir $NVM_DIR"
+  if [ -s "$NVM_DIR/nvm.sh" ]; then
+    echo "Running nvm"
+    . "$NVM_DIR/nvm.sh" # This loads nvm
+  else
+    set -e
```
Comment (jcague) [inline scripts/checkNvm.sh]: do we need this `set -e`here?

File: scripts/checkNvm.sh [hunk 6]
```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
 set -e
```
Comment (jcague) [inline scripts/checkNvm.sh]: The use of set -e in this file is a bit tricky IMO, can we just get rid of it if we don't to use it?

File: scripts/installUbuntuDeps.sh [hunk 7]
```diff
@@ -44,27 +46,33 @@ check_proxy(){
...
+    . $NVM_CHECK
+    nvm install
+  else
+    mkdir -p $LIB_DIR
+    install_nvm_node
+  fi
 }
 
 install_apt_deps(){
+
```
Comment (jcague) [inline scripts/installUbuntuDeps.sh]: Nit-picky comment: please remove this blank line

File: scripts/travisInstall.sh (hunk 2 above)
Comment (lodoyun) [inline scripts/travisInstall.sh]: :+1:

File: scripts/travisScript.sh (hunk 3 above)
Comment (lodoyun) [inline scripts/travisScript.sh]: :+1:

File: .travis.yml (hunk 1 above)
Comment (jcague) [inline .travis.yml]: :+1:

File: scripts/checkNvm.sh (hunk 5 above)
Comment (lodoyun) [inline scripts/checkNvm.sh]: Right :+1:

**Comment by mccob** [comment]
I think there is an issue.

Get this : 

$  ./licode/scripts/initBasicExample.sh
Checking dir /home/user/licode/build/libdeps/nvm
Running nvm
No .nvmrc file found

**Comment by lodoyun** [comment]
@mccob Is the .nvmrc file in /licode ?
Can you try running the installation script from licode/scripts (cd there before) and tell me if it works?

**Comment by mccob** [comment]
yes

 more .nvmrc
6.9.2

Maybe there is an error because I have use this https://github.com/ging/licode/issues/660 to solve another issue

**Pull request merged**